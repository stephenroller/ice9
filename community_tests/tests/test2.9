# test2 - make sure comments don't mess up syntax failure on empty file
