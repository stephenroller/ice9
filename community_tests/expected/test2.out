Loading file: generated/test2.tm
Number of instructions executed = 5
